sidebars.js

```diff
@@ -134,6 +134,40 @@ const sidebars = {
       items: ['migration/datadog/vector'],
     },
   ],
+},
+{ 
+  type: 'category',
+  label: 'Splunk Commands',
+  link: {
+    type: 'generated-index',
+  },
+  items: [
+ 'spl-docs/splunk-commands',
+{
+  type: 'category',
+  label: 'Evaluation Functions',
+  link: {
+    type: 'generated-index',
+    title: 'Evaluation Functions',
+    description: `Use evaluation functions to evaluate an expression on your events`,
+  },
+  items: ['spl-docs/evaluation-functions/overview', 'spl-docs/evaluation-functions/informational-functions', 'spl-docs/evaluation-functions/conversion-functions', 'spl-docs/evaluation-functions/time-functions'],
```

Comment:
nit: for this `items` line, you might want to have each item on a separate line (to reduce merge conflicts and make it slightly easier to rearrange items). But also you can ignore this if you want and keep it on one line

Reply:
addressed in the latest commit